shared/actions/teams.tsx

```diff
@@ -968,7 +981,12 @@ function* createChannel(_, action, logger) {
     }
 
     // Dismiss the create channel dialog.
-    if (Router2Constants.getVisibleScreen()?.routeName === 'chatCreateChannel') {
+    if (
+      // Auto generated from flowToTs. Please clean me!
+      (Router2Constants.getVisibleScreen() === null || Router2Constants.getVisibleScreen() === undefined
+        ? undefined
+        : Router2Constants.getVisibleScreen().routeName) === 'chatCreateChannel'
```

Comment:
I don't think we should merge in unconverted code like this, could be really unsafe